### DISCUSSION TITLE
Show us yours

### DISCUSSION THREAD

**Discussion by alex-code** [DCC-EX/EX-T3-WiFi#74] [Show and tell]
Thinking about the different types of 3D printers, materials and colours I thought I'd start a show us yours discussion where you can add a pic of your finished throttle for all to see 😊

**Comment by Alan-Lomax** [comment]
**OK .. I will take the bait.** 
Here are the  two I have built. Both builds were mostly as per instructions (the new instructions!). The only obvious difference is I used the knob that came with the encoder vs using the 3d printed one - which did not fit very well and I could not be bothered to rescale and reprint. Material is standard PLA and it worked fine.
**Note:** When threading in the screws do take care to gently rotate the screws back and forth when first cutting the threads. 

The black one (Which obviously still has the screen protector film on it) . 
![Black_Throttle_Finished](https://user-images.githubusercontent.com/87946633/227724136-dd8ace7e-da7e-4159-950b-65f290759a40.jpg)

The Red One
![Red_Throttle_Finished](https://user-images.githubusercontent.com/87946633/227724281-ce34a115-5843-4af0-a436-eae1507a3a7c.jpg)

And an Internal View When Under construction. (Here you can see the small change from the base instructions - I used a 90 degree male header on the encoder .. and the female connector has the wires attached. No particular reason except that was what I had available)
![Red_Throttle_UnderConstruction](https://user-images.githubusercontent.com/87946633/227724405-21981f79-8fe1-42ba-a203-d72a0e9fbd20.jpg)

**Reply by alex-code** [reply]
Nice, it is awkward with the 3D parts as tolerances will vary between each printer.

If the dial was loose I wonder if a bit of electrical tape on the encoder shaft could give a more snug fit.

**Reply by Alan-Lomax** [reply]
Unfortunately the dial was not loose it simply would not go on the shaft and I was reluctant to apply too much force and risk damaging the encoder. (and I had the OEM knob so I was quickly on to plan 'B'), Agreed about tolerances in this case a friction fit would be required (which is a 0.2 mm gap between male and female parts) and it was not there. I did not measure to find out exactly where the issue was.

**Reply by daveclifford** [reply]
Nice use ot two-tone  colors - reminds me of a '53 Chevy my uncle had ... (ha, ha)

How were the photos taken - with a phone??

Dave

**Reply by Alan-Lomax** [reply]
Re two tone colours - since I was printing two I decided to swap colours then just mixed and matched the parts. I wanted them to be distinct since likely they would be running different sets of locos during a session.  Yes the photos were taken with my "few years old iPhone". I used a 3d printed 'sweep stand' ( https://www.thingiverse.com/thing:3638751 ) with a sheet of black craft paper.

**Reply by daveclifford** [reply]
Thanks for the info. I had one of my cases printed commercially and the other my nephew printed. I can't really tell the difference in quality. 

You and I think much alike - with printing different colored cases.

Attached is the color wheel the printer sent me. You will have to wait for my pics.

![Color-Sample](https://user-images.githubusercontent.com/1976894/227811380-ac925031-3879-4363-b880-7f8b0e550258.jpg)

**Comment by Budman1758** [comment]
I have three of these throttles in the works. For the most part they are built as per instructions with the battery being the big difference. While looking for the 3000 mAh versions I cam upon some [3700 mAh versions](https://www.amazon.com/dp/B08HGSXYM1?psc=1&ref=ppx_yo2ov_dt_b_product_details). Measurements said they should fit so I ordered a 4 pack (much better price per). Only thing is they are somewhat thicker than the 2 & 3000 versions and was a very tight fit in the case. I also had to change the wires on the battery plug as they were bassakwards.  Seems like they could standardize that?????

I imported the case stl file into Tinkercad and made the case about 5mm taller so there was plenty of room for the bigger battery. If desired I can upload the stl file for all to use. Dont really know where to upload it so its available at [this link](https://www.catmakanic.com/Bodydeep.stl).

I had the same issue with the dial not fitting the encoder shaft. I scaled it up to 102% percent in Cura and that fixed that issue just fine. 

I like blue.....  :)

![PXL_20230405_053243882](https://user-images.githubusercontent.com/6570134/229990541-21d760bb-f648-437a-829a-99243f16f32a.jpg)
![PXL_20230405_053317658](https://user-images.githubusercontent.com/6570134/229990546-3ac2633b-fdb7-4744-8e91-1510a542bfb6.jpg)
![PXL_20230405_053326677](https://user-images.githubusercontent.com/6570134/229990547-3fa196ca-e721-46d2-a638-55e887031f86.jpg)
![PXL_20230405_053339012](https://user-images.githubusercontent.com/6570134/229990549-bf533743-9a58-4e50-9402-0f734a77f86d.jpg)

**Reply by alex-code** [reply]
Looks good, it is annoying that the batteries can be wired differently.
Should get some good running time from a 3700mAh battery.

**Reply by daveclifford** [reply]
Alex,

The LiPo batteries are sold for the RC (remote control) market (model cars and planes, etc.) and are wired (as the RC standard) opposite of what the IoT Group uses -  hence the continuing clash between the RC and IoT devotees.

I believe the check for correct battery polarity should be **both first and last steps** to do in assembling the throttle from the assembled parts. Please emphasise these steps!!

A LiPo battery fire is a bad thing to endure;  I have personally witnessed one and the fire damaged an 8x12 office - damage was estimated to be $12,000.

Dave

**Comment by LonelyFireWolf** [comment]
Heres mine!
If somebody want to make it here's link to my thread:
https://github.com/DCC-EX/EX-T3-WiFi/discussions/133
![IMG_20250405_224443](https://github.com/user-attachments/assets/4a4c285e-830b-45f1-8d57-5e08a520cdd7)
